hawkbit-dmf/hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/integration/AbstractAmqpServiceIntegrationTest.java

```diff
@@ -102,6 +110,17 @@ public void initListener() {
         }
     }
 
+    protected void waitUntilEventMessagesAreDispatchedToTarget(final EventTopic... eventTopics) {
+        createConditionFactory().untilAsserted(() -> {
+            assertThat(eventTopics).containsExactlyInAnyOrderElementsOf(replyToListener.getLatestEventMessageTopics());
```

Comment:
Shouldn't it be the other way round?
```suggestion
            assertThat(replyToListener.getLatestEventMessageTopics()).containsExactlyInAnyOrderElementsOf(eventTopics);
```